docs/en/magnetic_grip.md

```diff
@@ -0,0 +1,65 @@
+# Assembling and setting up the electromagnetic gripper
+
+The magnetic gripper can be assembled in various ways according to the wiring diagram.
+
+<img src="../assets/magnet_grip/scheme.jpg" width=300 class="zoom border center">
+
+The following is an example of assembling an electromagnetic capture circuit on a breadboard.
+
+> **Info** It is recommended to lay the wiring between the elements on the back side of the board (in the following images, the wiring is done over the diagram for illustrative purpose).
+
+1. Place the Schottky diode, 10K resistor, and transistor on the soldering board.
+
+    <img src="../assets/magnet_grip/magnet1.png" width=300 class="zoom border center">
+
+2. Solder the contacts on the other side of the board and bite off the remaining element legs.
+3. Connect the pins of the resistor and the two outer legs of the transistor.
+
+    <img src="../assets/magnet_grip/magnet2.png" width=300 class="zoom border center">
+
+4. Connect the center leg of the transistor and the leg of the Schottky diode (opposite to the gray marking strip).
+
+    <img src="../assets/magnet_grip/magnet3.png" width=300 class="zoom border center">
+
+5. Cut the required amount of magnetic grab wire and solder it to the pins of the Schottky diode.
+
+    <img src="../assets/magnet_grip/magnet4.png" width=300 class="zoom border center">
+
+6. Solder the *Dupont* -dad wires to the transistor and diode leg (red, black wires), and the *Dupont* -mama wire to the opposite transistor leg (white wire).
```

Comment:
Dupont "dad" and "mama"? WTF, @Alamoris.

Reply:
Uuups, fix